swarm/storage/localstore/localstore.go

```diff
@@ -284,9 +284,28 @@ func New(path string, baseKey []byte, o *Options) (db *DB, err error) {
 			return e, nil
 		},
 		EncodeValue: func(fields shed.Item) (value []byte, err error) {
-			return nil, nil
+			valueBuffer := []byte{}
```

Comment:
@janos could you please review this delta? this is what I came up with for push tags. I actually was thinking it might be cleaner to just have the name of the index changed and leave all the rest to the PR that I'm working on since this is a bit of a wider change. We could, however, leave this as it is. I don't think it would break anything or cause any harm

Reply:
@justelad I really like how you managed to make the most compact tags encoding. I had to validate it, and the result was TestDB_pushIndex_Tags that I pushed to this branch.